ray-operator/apis/ray/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -143,6 +143,21 @@ func (in *HeadGroupSpec) DeepCopy() *HeadGroupSpec {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *HeadInfo) DeepCopyInto(out *HeadInfo) {
+	*out = *in
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new HeadInfo.
+func (in *HeadInfo) DeepCopy() *HeadInfo {
+	if in == nil {
+		return nil
+	}
+	out := new(HeadInfo)
+	in.DeepCopyInto(out)
+	return out
+}
```

Comment:
Did we forget to run a CI step when we merged this PR: https://github.com/ray-project/kuberay/pull/468 ?

Reply:
Opened an issue to track so we don't forget about it
https://github.com/ray-project/kuberay/issues/720